src/Form/index.tsx

```diff
@@ -1683,18 +1683,29 @@ function Form({
           break;
         }
       } else if (type === ACTION_INVITE_COLLABORATOR) {
+        // Invited collaborators is a mixed list of emails and/or user group names (comma sep or array)
         const val = fieldValues[action.email_field_key];
-        const emails = toList(val, true);
-        const invalidEmail = emails.reduce((acc, email) => {
-          if (!validators.email(email)) return true;
+        if (!val) {
+          setElementError('Collaborators required');
+          break;
+        }
+        const invitees = toList(val, true);
+        const invalidInvitee = invitees.reduce((acc, invitee) => {
+          if (
+            !validators.email(invitee) &&
+            !session.user_groups.includes(invitee)
```

Comment:
instead of returning all user groups to frontend, this validation wodl be better done on the backend when sending the invite

Reply:
Ok, removed FE/lib validation.  No longer sending user groups from BE.